src/storekit.cs

```diff
@@ -797,9 +834,16 @@ interface SKProductStorePromotionController {
 	[DisableDefaultCtor] // Not specified but very likely
 	interface SKStoreReviewController {
 
+		[Deprecated (PlatformName.iOS, 9, 0, message : "Use the 'RequestReview (UIWindowScene windowScene)' API instead.")]
 		[Static]
 		[Export ("requestReview")]
 		void RequestReview ();
+
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[NoWatch, NoTV, iOS (14,0), Mac (11,0)]
```

Comment:
`[Mac (11,0)]` is incorrect as `UIWindowScene` does not exists in macOS
This compiled because you defined it to be `NSObject` but that won't work at runtime